app/views/sitemap/robots.text.erb

```diff
@@ -1,3 +1,4 @@
+User-agent: *
```

Comment:
@pmcostello is this to block *all* crawlers from meandering through the Frontend site as per https://support.google.com/webmasters/answer/1061943?hl=en

Reply:
yeah, the default for the robots file is to allow * crawler bots but unless the User-Agent is explicitly declared the Disallow directives are being ignored